include/octo-logger-cpp/context-info.hpp

```diff
@@ -32,7 +32,7 @@ class ContextInfo
     ContextInfoType context_info_;
 
   public:
-    virtual ~ContextInfo() = default;
+    ~ContextInfo() = default;
```

Comment:
requested by Denis before

Reply:
Spoke to Denis and marked the class as `final`